src/cpp/dds/topic/DataWriterImpl.cpp

```diff
@@ -580,12 +591,15 @@ void DataWriterImpl::InnerDataWriterListener::on_liveliness_lost(
         fastrtps::rtps::RTPSWriter* /*writer*/,
         const fastrtps::LivelinessLostStatus& status)
 {
+    LivelinessLostStatus liveliness_status;
```

Comment:
Probably `fastrpts::LivelinessLostStatus` should be removed in favor of `fastdds::dds::LivelinessLostStatus`.

Reply:
Changed